services/movie.py

```diff
@@ -0,0 +1,43 @@
+from django.db.models import QuerySet
+
+from db.models import Movie
+
+
+def get_movies(
+        genres_ids: list[int] = None,
+        actors_ids: list[int] = None
+) -> QuerySet:
+
+    movies_set = Movie.objects.all()
+    if genres_ids and actors_ids:
+        movies_set = movies_set.filter(
+            genres__id__in=genres_ids,
+            actors__id__in=actors_ids
+        )
+    if genres_ids and not actors_ids:
+        movies_set = movies_set.filter(genres__id__in=genres_ids)
+    if actors_ids and not genres_ids:
+        movies_set = movies_set.filter(actors__id__in=actors_ids)
+
+    return movies_set
+
+
+def get_movie_by_id(movie_id: int) -> Movie:
+    return Movie.objects.get(id=movie_id)

```

Comment:
Consider handling the case where a movie with the given `movie_id` does not exist. Using `get()` can raise a `DoesNotExist` exception if no object is found. You might want to handle this exception or use `get_object_or_404` if this is part of a Django view.